app/models/concerns/hmis/structure/enrollment.rb

```diff
@@ -8,8 +8,15 @@ module HMIS::Structure::Enrollment
   extend ActiveSupport::Concern
   include ::HMIS::Structure::Base
 
+  included do
+    self.hud_key = :EnrollmentID
+    self.conflict_target = [:data_source_id, '"EnrollmentID"', '"PersonalID"']
```

Comment:
These should be `connection.quote_column_name(e_t[:EnrollmentID])` or something similar.